examples/kubernetes/manifests/portus.yml

```diff
@@ -0,0 +1,87 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: portus
+  labels:
+    app: portus
+    tier: registry
+spec:
+  type: NodePort
+  # type: LoadBalancer
+  ports:
+  - port: 3000
+    targetPort: 3000
```

Comment:
Maybe I should add documentation on this. It's stated on the official Docker image of Portus though.